pkg/controller/spinnakervalidating/spinnakervalidating_controller.go

```diff
@@ -77,6 +83,36 @@ func Add(m manager.Manager) error {
 	return as.Register(validatingWebhook)
 }
 
+func getSpinnakerServiceRule() admissionregistrationv1beta1.RuleWithOperations {
+	gv := v1alpha1.SchemeGroupVersion
```

Comment:
This needs to be pluggable the same way that we use `SpinnakerServiceInterface` on the service controller.